### PR TITLE
Add P1AM-200 Board

### PR DESCRIPTION
**Pull request by AdamCummick**

### CODE REVIEW

File: ports/atmel-samd/boards/p1am_200/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+USB_VID = 0x1354
+USB_PID = 0x4004
+USB_PRODUCT = "P1AM-200 Circuitpython"
```
Comment (dhalbert) [inline ports/atmel-samd/boards/p1am_200/mpconfigboard.mk:3]:
```suggestion
USB_PRODUCT = "P1AM-200 CircuitPython"
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
Hi -a couple of minor changes.

File: ports/atmel-samd/boards/p1am_200/mpconfigboard.h [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#define MICROPY_HW_NEOPIXEL (&pin_PC24)
+
+// These are pins not to reset.
+// QSPI Data pins
+#define MICROPY_PORT_A (PORT_PA08 | PORT_PA09 | PORT_PA10 | PORT_PA11)
+// QSPI CS, and QSPI SCK
+#define MICROPY_PORT_B (PORT_PB10 | PORT_PB11)
+#define MICROPY_PORT_C (0)
+#define MICROPY_PORT_D (0)
+
```
Comment (dhalbert) [inline ports/atmel-samd/boards/p1am_200/mpconfigboard.h:17]: This is no longer needed. See https://github.com/adafruit/circuitpython/pull/5007.
```suggestion
```

**Review by dhalbert — APPROVED** [review]

**Comment by AdamCummick** [comment]
Thanks, Dan!

**Comment by dhalbert** [comment]
File a PR for https://github.com/adafruit/circuitpython-org to describe the board too. See https://learn.adafruit.com/how-to-add-a-new-board-to-the-circuitpython-org-website for details.

**Pull request merged**